### PR TITLE
Update index.html to remove link to old DIST pages

### PR DESCRIPTION
**Pull request by MarkusPiotrowski**
...which are now re-directed to the Sphinx doc pages

### CODE REVIEW

File: 1.74/api/index.html [hunk 1]
```diff
@@ -232,4 +230,4 @@ <h1>Indices and tables<a class="headerlink" href="#indices-and-tables" title="Pe
    
 
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (peterjc) [inline 1.74/api/index.html:233]: If you're adding this, then we need an opening ``<html>`` tag too. Maybe just go for the minimal change?

Looks like this bug in Sphinx has been fixed, our 1.78 docs has ``<html class="writer-html5" lang="en" >`` near the start.

**Review by peterjc — CHANGES_REQUESTED** [review]
Do you want to change https://github.com/biopython/docs/blob/gh-pages/1.75/api/index.html too?

File: 1.74/api/index.html (hunk 1 above)
Comment (MarkusPiotrowski) [reply 1.74/api/index.html:233]: I guess this closing `<\html>` was automatically added because the one which was there before had no newline.

**Comment by MarkusPiotrowski** [comment]
Sorry, I was interrupted during committing. Did the same changes now in 1.75 and 1.76 documentation.

File: 1.74/api/index.html (hunk 1 above)
Comment (MarkusPiotrowski) [reply 1.74/api/index.html:233]: So, I did not add it, it was replaced.

Comment (peterjc) [reply 1.74/api/index.html:233]: Ah - GitHub is trying to tell me there was no trailing line symbol. I see the opening ``<html>`` now hidden in some magic comma if statements for Internet Explorer.

Sorry, false alarm.

**Review by peterjc — APPROVED** [review]

**Comment by MarkusPiotrowski** [comment]
@peterjc I don't have the rights to merge here, so you need to do this

**Comment by peterjc** [comment]
Permissions was an oversight, this still had the defaults with just me. Fixed to use the same teams as the main repo.

**Pull request merged**